docs/001_develop/03_client-capabilities/008_interaction/slider.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Slider'
+sidebar_label: 'Slider'
+id: client-interaction-slider
+keywords: [interaction, interactive, input, slider, form, data]
+tags:
+- interaction
+- interactive
+- input
+- slider
+- form
+- data
+sidebar_position: 8
+---
+
+import SliderDemo from '../../../../examples/ui/client-capabilities/interaction/slider.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Slider
+
+Use cases:
+* Interact with data visualization
+* Input for numeric values
+* Dynamic filters
```

Comment:
add a point about the slider labels.

Anyone know what else you can do with this component?